pages/about/groups/task-forces/wcag2ict/work-statement.md

```diff
@@ -61,7 +63,7 @@ The primary effort of this Task Force is to describe how to interpret WCAG 2.x a
 
 To efficiently develop the Working Group Note, work may be broken down into sub-groups, under the coordination of the Task Force facilitator(s) following established AG WG procedures for sub-groups.
 
-The work will be focused in stages to ensure WCAG2ICT content is available in time for use in upcoming updates to regulations and standards (like the EN 301 549) that apply WCAG to non-web technology. This means prioritizing Level A and AA criteria and known issues in the existing Note. WCAG2ICT could potentially need to be published before addressing Level AAA due to regulatory schedule needs. If publication is needed prior to addressing Level AAA criteria, another update to the WCAG2ICT Note may be necessary.
+The work will be focused in stages to ensure WCAG2ICT content is available in time for use in upcoming updates to regulations and standards (like the EN 301 549) that apply WCAG to non-web technology. This means prioritizing Level A and AA criteria and known issues in the existing Note. WCAG2ICT was published on 4 November 2024 before addressing Level AAA, due to regulatory schedule needs. The next update to WCAG2ICT will address Level AAA success criteria, with republication of the WCAG2ICT Note once they have been incorporated and gone through the W3C review and publication process.
```

Comment:
"due to regulatory needs" seems a bit ambitious.  Did any regulators/legislators make the ask?

Maybe "due to need for W3C to be proactive to regulation in timely fashion"?

Reply:
This exact statement was in the original work statement and left unchanged, just changing to add in the date of what was already done.

Reply:
I am okay with original language.